package/kernel/linux/modules/leds.mk

```
@@ -16,6 +16,27 @@ define KernelPackage/leds-gpio
   AUTOLOAD:=$(call AutoLoad,60,leds-gpio,1)
 endef
 
+
+define KernelPackage/led-group-multicolor
+	 SUBMENU:=$(LEDS_MENU)
+	 TITLE:=LEDs group multi-color support
+	 KCONFIG:= \
+	     CONFIG_LEDS_CLASS_MULTICOLOR=y \
```

Please build this as a module too:
```suggestion
	     CONFIG_LEDS_CLASS_MULTICOLOR \
```